### DISCUSSION TITLE
First contribution ideas

### DISCUSSION THREAD

**Discussion by mattl** [foocorp/hacienda#108] [Future enhancements]
https://github.com/foocorp/documentation/wiki/First-contribution-to-Libre.fm

**Comment by inan-hira** [comment]
Do you plan to add issue templates, @mattl? It might be good to standardize the bug report tickets.

**Comment by mattl** [comment]
Yes, I'd like to add those

**Comment by mattl** [comment]
@inan-hira We're now using templates and discussions :)
